go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/cosmos/ledger-cosmos-go v0.11.1
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/mock v1.3.1-0.20190508161146-9fa652df1129
+	github.com/golang/protobuf v1.3.2
```

Comment:
Why is golang/protobuf required? This likely means there's a directive needed to make a golang/protobuf type to a gogo/protobuf type

Reply:
Because of our dependency on

```
import "google/protobuf/timestamp.proto";
import "google/protobuf/duration.proto";
```

e.g.

```
message Commission {
  option (gogoproto.equal) = true;
  option (gogoproto.goproto_stringer) = false;

  CommissionRates commission_rates = 1
      [ (gogoproto.embed) = true, (gogoproto.nullable) = false ];
  google.protobuf.Timestamp update_time = 2 [
    (gogoproto.nullable) = false,
    (gogoproto.stdtime) = true,
    (gogoproto.moretags) = "yaml:\"update_time\""
  ];
}

gogo proto itself depends on `google/protobuf`. Does gogo proto itself provide these? If so, what advantage is there with one over the other? 
```

Reply:
You need this in protocgen.sh:

```go
protoc -I=. -I=$GOPATH/src -I=$GOPATH/src/github.com/gogo/protobuf/protobuf --{binary}_out=\
Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/struct.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,\
Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types:. \
myproto.proto
```

See https://github.com/gogo/protobuf#more-speed-and-more-generated-code